likes_lookup_test.go

```diff
@@ -0,0 +1,231 @@
+package gotwtr_test
+
+import (
+	"context"
+	"io"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/sivchari/gotwtr"
+)
+
+func Test_lookUpUsersWhoLiked(t *testing.T) {
+	type args struct {
```

Comment:
t.Parallel()

Reply:
Is your intention to add `t.Parallel()` to the subtest? If so, I have already added it.
If your intention is to add `t.Parallel()` to top-level test, I think it is unnecessary because in the other top-level tests, `t.Parallel()` is not present.

Reply:
> I think it is unnecessary because in the other top-level tests, t.Parallel() is not present.

If this is reason, it's not good reason.
I'll add `t.Parallel()` to all top-level-test, soon.
If you have another reason, please describe.

Reply:
I thought it would be more appropriate to add `t.Parallel()` to all top-level-tests together after merging this PullRequest, intead of adding `t.Parallel()` to a top-level-test in this PullRequest, in terms of the semantics of that commit that you try to do.

Reply:
above `commit that you try to do` is intended for `I'll add t.Parallel() to all top-level-test, soon.`

Reply:
> I thought it would be more appropriate to add t.Parallel() to all top-level-tests together after merging this PullRequest, intead of adding t.Parallel() to a top-level-test in this PullRequest, in terms of the semantics of that commit that you try to do.

I don't agree with above comment.
A future fix is ​​to add to tests that don't currently support `t.Parallel ()`.
The problem this time is that there is no `t.Parallel ()` at the top level of this PR.
In other words, it is completely different from intentionally not supporting `t.Parallel ()` this time.